### ISSUE TITLE
No automatic backups through cron job.

### ISSUE THREAD

**Issue by AS-dh** [pbek/ownbackup#11]
After successful installation only manual backups work. No automatic backups through cron job. 
This is for nextcloud 11.0.1. with app installed from the App list in Admin. The cron job is set up and running. There is an entry in oc_jobs with "0" in last_run and the date of the one and only manual in last_checked. Have stopped and started the cron job and also put the whole site in maintenance mode and back to no avail.
Will very much appreciate any help as backups need to run automatically to make sense...

**Comment by pbek** [comment]
I'm sorry, I never came across this problem.

**Comment by AS-dh** [comment]
Ok, it was my fault: The cron.php script was not running properly and I didn't notice. For the sake of those who might run into a similar situation here is a more detailed explanation of what happened:
I have my nextcloud running on a shared hoster (all-inkl) that provides cronjob functionality through their management console (https://kas.all-inkl.com).  Shortly before I installed ownbackup I'd changed the cron type in the nextcloud admin server settings from Webcron to Cron because I incorrectly assumed that the latter was the right one.  It was only after calling the https://nextcloud.mydomain.com/cron.php directly that I realized it was producing an error and not running the jobs listed in oc_jobs.
